### ISSUE TITLE
Dependency upgrade breaks Spectacle

### ISSUE THREAD

**Issue by anfedorov** [sourcey/spectacle#122]
Not sure whether it's to do with my specific file, but this used to work before a recent dependency upgrade:

```
$ spectacle --target-dir docs/dev.sig.ma/public/docs docs/dev.sig.ma/REST-API_v1_swagger.yaml
Running "sass:foundation_scss" (sass) task
WARNING: Note, that $header-sizes has been replaced with $header-styles. $header-sizes still works, but it is going to be depreciated.
Backtrace:
	../../../../usr/local/lib/node_modules/spectacle-docs/app/vendor/foundation/scss/typography/_base.scss:77, in function `build_from_header-sizes`
	../../../../usr/local/lib/node_modules/spectacle-docs/app/vendor/foundation/scss/typography/_base.scss:91


Running "concat:foundation_css" (concat) task
File docs/dev.sig.ma/public/docs/stylesheets/foundation.css created.

Running "sass:scss" (sass) task
WARNING: Note, that $header-sizes has been replaced with $header-styles. $header-sizes still works, but it is going to be depreciated.
Backtrace:
	../../../../usr/local/lib/node_modules/spectacle-docs/app/vendor/foundation/scss/typography/_base.scss:77, in function `build_from_header-sizes`
	../../../../usr/local/lib/node_modules/spectacle-docs/app/vendor/foundation/scss/typography/_base.scss:91


Running "concat:css" (concat) task
File docs/dev.sig.ma/public/docs/stylesheets/spectacle.css created.

Running "cssmin:minify" (cssmin) task
>> 2 files created. 146.33 kB → 113.2 kB

Running "concat:js" (concat) task
File docs/dev.sig.ma/public/docs/javascripts/spectacle.js created.

Running "uglify:js" (uglify) task
>> 1 file created.

Running "clean:html" (clean) task
>> 0 paths cleaned.

Running "compile-handlebars:compile" (compile-handlebars) task
Task error: Error: marked(): input parameter is of type [object String], string expected
    at marked (/usr/local/lib/node_modules/spectacle-docs/node_modules/marked/lib/marked.js:1242:11)
    at Object.markdown (/usr/local/lib/node_modules/spectacle-docs/app/lib/common.js:20:17)
    at String.module.exports (/usr/local/lib/node_modules/spectacle-docs/app/helpers/md.js:13:21)
    at eval (eval at createFunctionContext (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:254:23), <anonymous>:4:97)
    at Object.prog [as fn] (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/runtime.js:221:12)
    at String.module.exports (/usr/local/lib/node_modules/spectacle-docs/app/helpers/ifeq.js:19:20)
    at eval (eval at createFunctionContext (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:254:23), <anonymous>:8:70)
    at Object.prog [as inverse] (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/runtime.js:221:12)
    at String.module.exports (/usr/local/lib/node_modules/spectacle-docs/app/helpers/ifin.js:5:18)
    at eval (eval at createFunctionContext (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:254:23), <anonymous>:5:88)
    at prog (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/runtime.js:221:12)
    at execIteration (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:51:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:80:11)
    at eval (eval at createFunctionContext (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:254:23), <anonymous>:12:31)
    at prog (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/runtime.js:221:12)
    at execIteration (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:51:19)

Running "predentation" task
Task error: { Error: ENOENT: no such file or directory, open '/var/folders/mk/zfrtsj613vs9h1hzmzyt908w0000gn/T/spectacle-34695gYXy678pFUiS/index.html'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at Object.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/index.js:90:23)
    at Object.thisTask.fn (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/grunt/task.js:82:16)
    at Object.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:301:30)
    at Task.runTaskFn (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:251:24)
    at Task.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:300:12)
    at Task.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:230:9)
    at Task.runTaskFn (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:257:7)
    at Task.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:300:12)
    at Task.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:230:9)
    at Task.runTaskFn (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:254:9)
    at Task.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:300:12)
    at Task.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:230:9)
    at Task.runTaskFn (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:254:9)
    at Task.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/node_modules/grunt/lib/util/task.js:300:12)
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/var/folders/mk/zfrtsj613vs9h1hzmzyt908w0000gn/T/spectacle-34695gYXy678pFUiS/index.html' }

Running "prettify:index" (prettify) task
>> Destination not written because dest file was empty.
All tasks complete
Unhandled rejection Error: marked(): input parameter is of type [object String], string expected
    at marked (/usr/local/lib/node_modules/spectacle-docs/node_modules/marked/lib/marked.js:1242:11)
    at Object.markdown (/usr/local/lib/node_modules/spectacle-docs/app/lib/common.js:20:17)
    at String.module.exports (/usr/local/lib/node_modules/spectacle-docs/app/helpers/md.js:13:21)
    at eval (eval at createFunctionContext (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:254:23), <anonymous>:4:97)
    at Object.prog [as fn] (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/runtime.js:221:12)
    at String.module.exports (/usr/local/lib/node_modules/spectacle-docs/app/helpers/ifeq.js:19:20)
    at eval (eval at createFunctionContext (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:254:23), <anonymous>:8:70)
    at Object.prog [as inverse] (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/runtime.js:221:12)
    at String.module.exports (/usr/local/lib/node_modules/spectacle-docs/app/helpers/ifin.js:5:18)
    at eval (eval at createFunctionContext (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:254:23), <anonymous>:5:88)
    at prog (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/runtime.js:221:12)
    at execIteration (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:51:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:80:11)
    at eval (eval at createFunctionContext (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:254:23), <anonymous>:12:31)
    at prog (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/runtime.js:221:12)
    at execIteration (/usr/local/lib/node_modules/spectacle-docs/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:51:19)

```

**Comment by auscaster** [comment]
Can you please send me a copy of the breaking spec file to auscaster@gmail.com?

**Comment by auscaster** [comment]
I was able to reproduce the error and this one is fixed in 1.0.1 :)

**Comment by anfedorov** [comment]
Thanks. Sorry I missed your earlier request.
